mhs/common/mhs_common/workflow/__init__.py

```diff
@@ -10,14 +10,18 @@
 from mhs_common.workflow.common import CommonWorkflow
 from mhs_common.workflow.intermediary_reliable import IntermediaryReliableWorkflow
 from mhs_common.workflow.synchronous import SynchronousWorkflow
+from mhs_common.workflow.sync_async import SyncAsyncWorkflow
 
 ASYNC_EXPRESS = 'async-express'
 ASYNC_RELIABLE = 'async-reliable'
 FORWARD_RELIABLE = 'forward-reliable'
 SYNC = 'sync'
+SYNC_ASYNC = 'sync-async'
 
 
-def get_workflow_map(party_key: str = None, persistence_store: persistence_adaptor.PersistenceAdaptor = None,
+def get_workflow_map(party_key: str = None,
+                     persistence_store: persistence_adaptor.PersistenceAdaptor = None,
```

Comment:
Think this can be renamed to something more specific.

Reply:
`persistence store`? I've updated it to `work_description_store` to reflect what it actually contains, I've updated the inbound handler and main files to make it consistent

Reply:
I've also updated the `queue_adaptor` parameter because was the same as the module name we are importing: `from comms import queue_adaptor`